src/components/geolocalisation/providers/mapbox.js

```diff
@@ -307,6 +307,15 @@ const leafletMap = (services) => {
                 if (shouldUpdateZoom) {
                     currentZoomLevel = map.getZoom();
                 }
+                $('#map-zoom-to-setting').val(map.getZoom());
+            });
+
+            map.on('dragend', function () {
+                var LngLat = map.getCenter();
+                var arr= [];
+                arr.push(String(LngLat['lat']));
+                arr.push(String(LngLat['lng']));
+               $('#map-position-to-setting').val('["'+LngLat['lat']+'","'+LngLat['lng'] +'"]');
```

Comment:
why using such complex serialization (brackets, quotes...) which will need complex code to unserialize ?
just make `lat + ';' + lng`

Reply:
the brackets and quotes are needed by mapbox  to read the lat and lng (a string but not an array ), the format should be like below  for example : 

["17.03","121.665"]

Reply:
ok I see, "map-xxx" is for "mapbox", not for generic "mapping" to setting.
A bit crappy code, but since it runs ok, let's not rewrite everything.
ok to merge